app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.parkingsystemjava">
+    <uses-permission android:name="android.permission.SET_WALLPAPER"/>
```

Comment:
why do you need this permission?

Reply:
I thought I needed it for the bottom, I already deleted it